benchmark/util/style-text.js

```diff
@@ -29,6 +30,8 @@ function main({ messageType, format, validateStream, n }) {
       break;
   }
 
+  process.env.NO_COLORS = noColors;
```

Comment:
Note that this line has no impact on the code because of:

- https://github.com/nodejs/node/pull/58072
- #58074